adaptors/hybrisproximityadaptor/hybrisproximityadaptor.cpp

```diff
@@ -108,7 +107,7 @@ void HybrisProximityAdaptor::sendInitialData()
            ProximityData *d = buffer->nextSlot();
 
            d->timestamp_ = Utils::getTimeStamp();
-           d->withinProximity_ = lastNearValue;
+           d->withinProximity = lastNearValue;
            d->value_ = lastNearValue ? 10 : 0;
```

Comment:
After the type change lastNearValue can be negative if ioctl probe fails -> is it ok that value gets assigned to 10?